volatility3/cli/__init__.py

```diff
@@ -366,6 +376,30 @@ def location_from_file(cls, filename: str) -> str:
                 raise ValueError(f"File does not exist: {filename}")
         return parse.urlunparse(single_location)
 
+    def load_system_defaults(self, filename: str) -> Tuple[List[Tuple[int, str]], Dict[str, Any]]:
+        """Modify the main configuration based on the default configuration override"""
+        # Build the config path
+        default_config_path = os.path.join(os.path.expanduser("~"), ".config", "volatility3", filename)
+        if sys.platform == 'win32':
+            default_config_path = os.path.join(os.environ.get("APPDATA", os.path.expanduser("~")), "volatility3",
+                                               filename)
+
+        delayed_logs = []
+
+        # Process it if the files exist
+        if os.path.exists(default_config_path):
+            result = json.load(open(default_config_path, 'rb'))
```

Comment:
What do you think of this way? :)

```suggestion
            with open(default_config_path, 'rb') as config_json:
                result = json.load(config_json)
```

Reply:
Yeah, I like it.  I think I even prefer it, thanks!  5;)